docs/DeveloperGuide.md

```diff
@@ -689,81 +642,77 @@ _Customization of Contacts_
 
       Use case resumes at step 2.
 
-* 3b. The given date(s) do not match the valid date format.
-
-    * 3b1. SociaLite shows an error message.
+* 3b. The specified contact is not pinned. 
 
-      Use case resumes at step 2.
+  Use case ends.
 
 <br/>
 
-***Help Guide & Exiting***
-
-**Use Case 15: View User Guide**
+**Use Case 15: Add remark for a specific contact**
 
 *MSS*
 
-1.  User requests to view User Guide
-2.  SociaLite displays a link to User Guide and instructions to obtain in-app guidance for five selected commands
+1.  User requests to **list contacts (UC02)** or **find contact (UC03)**
+2.  SociaLite returns a list of contacts according to the UC called
+3.  User specifies a remark to be added to the specified contact's entry
+4.  SociaLite adds a remark to the specified contact's entry
 
     Use case ends.
 
-<br/>
-
-**Use Case 16: View in-app guidance for selected commands**
-
-*MSS*
+*Extensions*
 
-1.  User requests to view in-app guidance for one out of five selected commands
-2.  SociaLite returns an overview and quick guide of the command given as input
+* 2a. The list is empty.
 
-    Use case ends.
+  Use case ends.
 
-*Extensions* 
+* 3a. The given index is invalid.
 
-* 1a. The keyword given as input is invalid.
+    * 3a1. SociaLite shows an error message.
 
-    * 1a1. SociaLite launches HelpWindow for **User Guide (UC15)** by default
-    
-        Use case ends.
+      Use case resumes at step 2.
 
 <br/>
 
-**Use Case 17: Purge contacts stored in SociaLite**
+**Use Case 16: View User Guide**
 
 *MSS*
 
-1.  User requests to clear all existing contacts (from demo version)
-2.  SociaLite clears specimen data from memory
+1.  User requests to view User Guide
+2.  SociaLite displays a link to User Guide and instructions to obtain in-app guidance for valid commands
 
     Use case ends.
 
 <br/>
 
-**Use Case 18: Exit application**
+**Use Case 17: View in-app guidance for selected commands**
 
 *MSS*
 
-1.  User types in command to exit application
-2.  SociaLite closes
+1.  User requests to view in-app guidance for valid commands
+2.  SociaLite returns an overview and quick guide of the command given as input
 
     Use case ends.
 
+*Extensions*
+
+* 1a. The keyword given as input is invalid.
 
-***Ease of Accessibility***
+    * 1a1. SociaLite launches HelpWindow for **User Guide (UC16)** by default
 
-**Use Case 19: Finding a contact in the list**
+      Use case ends.
 
-1. User types in the 'find' command with the names, tags and/or handle types to search for 
-2. Socialite shows all contacts matching all given input keywords in the UI, and the number of contacts found.
+<br/>
 
-    Use case ends.
+**Use Case 18: Exit application**
 
-*Extensions*
-* 2a. No contacts match all given keywords, blank list is returned
+*MSS*
+
+1.  User types in command to exit application
+2.  SociaLite closes
 
     Use case ends.
 
+
 *{More to be added}*
```

Comment:
Haha do we still need the more to be added since this is the last iteration?

Reply:
HAHA good point! maybe @david-eom can delete this line in his PR for the DG later?

Reply:
Alright! Will do for all the `{to be added}` stuff later in my PR!

Reply:
Thanks!